app/views/ReportView/components/PatientInformation/index.tsx

```diff
@@ -40,6 +41,11 @@ const PatientInformation = ({
   useEffect(() => {
     if (report?.ident) {
       const getData = async () => {
+        const appendicesResp = await api.get(`/reports/${report.ident}/appendices`).request();
+        let biopsyCollectionDate;
+        if (appendicesResp.sampleInfo) {
+          biopsyCollectionDate = appendicesResp.sampleInfo[0]['Collection Date'];
```

Comment:
is the API setup so that the first sampleInfo is always going to be what you're looking for? or is this a trival thing

Reply:
Good question! Sample Info sometimes has both "Tumor" and "Constitutional" sample collection date ("Tumor" always the first one), but sometimes only has "Constitutional" sample collection date. Will double check with Eleanor, when she's back.

Reply:
I've pinged clininfo on DEVSU-2333 to confirm but I think it'll be the tumour sample.

Reply:
Ok, yes it'll be the tumour sample collection date. If we don't have that, leave the field blank.

Reply:
Is there a way to check whether this is the tumour sample or not?